ping.go

```diff
@@ -6,7 +6,12 @@ import (
 
 type HealthModule struct{}
 
-func (m HealthModule) Register(e *gin.Engine) {
+func (m HealthModule) Register(g *gin.RouterGroup) {
+	g.GET("/ping", m.ping)
+	g.GET("/health", m.health)
+}
+
+func (m HealthModule) DeprecatedRegister(e *gin.Engine) {
```

Comment:
Would be nice with a comment here saying why it's deprecated and when it will potentially be removed. I feel that v5 is on the horizon, so maybe remove it in v6?